source/deployment/sso-saml.rst

```diff
@@ -0,0 +1,113 @@
+SAML Single-Sign-On (E20)
+-------------------------
+
+Overview
+~~~~~~~~
+
+SAML Single-Sign-On integration offers the following benefits:
+
+-  **Single-sign-on.** Users can sign-in to Mattermost with their SAML
+   credentials.
+-  **Centralized identity management.** Mattermost accounts can display
+   user information from SAML, such as first and last name, email and
+   username.
+-  **Automatic account provisioning.** New Mattermost user accounts are
+   automatically created the first time a user signs in with their SAML
+   credentials on the Mattermost server.
+
+Note that the standard SAML specification doesn't support a "lazy sync"
+option like AD/LDAP - instead, the SAML system is called every time the
+user authenticates in to Mattermost. Therefore, a user who has been
+deactivated in their identity provider will have access to Mattermost
+until their session expires and they need to log in again, or until
+`they are also deactivated in
+Mattermost <https://docs.mattermost.com/deployment/on-boarding.html#common-tasks>`__.
+
+Mattermost officially supports Okta and Microsoft ADFS as the identity
+providers (IDPs), please see links below for more details on how to
+configure SAML with Okta and Microsoft ADFS. If you'd like, you may also
+try configuring SAML for a custom IDP.
+
+`Configure SAML with Okta <http://docs.mattermost.com/deployment/sso-saml-okta.html>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+`Configure SAML with Microsoft ADFS <http://docs.mattermost.com/deployment/sso-saml-adfs.html>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pre-installation
+~~~~~~~~~~~~~~~~
+
```

Comment:
This section of the doc is confusing. The links look like titles instead of links.

![image](https://cloud.githubusercontent.com/assets/20325173/21818346/4ed9a31c-d71c-11e6-8311-95d7235de007.png)

Shouldn't the Pre-installation section come before the Okta and ADFS install sections?

Also, if using the monolith style, that content be included in-line.